src/utils/ipfs.js

```diff
@@ -16,6 +16,11 @@ function multihashToString(multihash) {
     return multihashes.toB58String(multihash);
 }
 
+function multihashToBuffer(multihash) {
+    if (Buffer.isBuffer(multihash)) return multihash;
+    return multihashes.fromB58String(multihash);
```

Comment:
Should we catch error here too?

Reply:
I think we should be careful here to handle bad data. Since we will be getting all sorts of potentially bad data provided as input to 'readReviewRecord', we should do some defensive programming and be sure our service node or wallets don't crash in case the data provided is in bad format, or incomplete.

Reply:
I'd even add we should write some tests with bad data provided as review multihash to read. Or bad data to write in the storeReviewRecord.

Reply:
I agree, I'll update this function to check whether whatever is passed is an actual multihash and also test failure cases

Reply:
I checked and it looks like I did handle errors when responding to requests written on the chlu pubsub topic, so service nodes should gracefully handle bad requests already